src/main/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositApp.scala

```diff
@@ -117,7 +119,15 @@ object SplitMultiDepositApp {
       permissions = configuration.properties.getString("deposit.permissions.access"),
       group = configuration.properties.getString("deposit.permissions.group")
     )
+    val ffprobe = {
+      val ffProbePath = configuration.properties.getString("audio-video.ffprobe")
```

Comment:
Would it make sense to do the thing below? That way you would properly catch a null pointer.

```scala
Option(configuration.properties.getString("audio-video.ffprobe"))
  .map(ffProbePath => {
    val exeFile = File(ffProbePath)
    require(exeFile isRegularFile, s"Ffprobe at $exeFile does not exist or is not a regular file")
    require(exeFile isExecutable, s"Ffprobe at $exeFile is not executable")
    FfprobeRunner(exeFile)
  })
  .getOrElse { throw new IllegalArgumentException("Missing configuration for ffprobe") }
```

Reply:
Besides, you're throwing an exception here (either explicitly as I proposed above, or implicitly using `require`), which is not documented in the type. Would it really hurt the code if we would change the signature to return `Try[SplitMultiDepositApp]`?

Reply:
I am not sure either way. I'll pass on this one, just to limit the scope of this PR. Maybe we should think about this as part of a refactoring of all the error handling in this project.